janus_core/helpers/post_process.py

```diff
@@ -270,17 +274,24 @@ def compute_vaf(
 
     vafs /= n_steps - np.arange(n_steps)
 
+    lags = np.arange(n_steps) * time_step
+
     if fft:
         vafs = np.fft.fft(vafs, axis=0)
-
-    vafs = [
-        np.average([vafs[used_atoms[i]] for i in atoms], axis=0)
-        for atoms in filter_atoms
-    ]
+        lags = 1. / lags
```

Comment:
Does this need to be 
```suggestion
        lags = 1. / reversed(lags)
```

Reply:
Presumably the vafs would also need to be reversed, so it's probably fine without?

Reply:
Presumably the vafs would also need to be reversed, so it's probably fine without?